### PR TITLE
feat(890): settings - warnings, tooltips, hide from non-admin

### PR DESCRIPTION
**Pull request by antkiewiczk**

### CODE REVIEW

File: apps/editor/src/app/base.tsx [hunk 1]
```diff
@@ -478,8 +478,7 @@ export function Base({children}: BaseProps) {
                     </PermissionControl>
                   </Nav.Menu>
                 </PermissionControl>
-                <PermissionControl
-                  qualifyingPermissions={['CAN_GET_SETTINGS', 'CAN_UPDATE_SETTINGS']}>
+                <PermissionControl qualifyingPermissions={['CAN_UPDATE_SETTINGS']}>
```
Comment (Itrulia) [inline apps/editor/src/app/base.tsx:481]: Is that on purpose that the CAN_GET_SETTINGS isn't needed anymore?

Comment (Itrulia) [reply apps/editor/src/app/base.tsx:481]: Then we should remove CanGetSettings from the editor's permission instead. There's other places (like the actual component) where that permission is also used.

**Review by Itrulia — CHANGES_REQUESTED** [review]
CanGetSettings should be removed from the editor's permission instead

**Review by michael-scheurer — CHANGES_REQUESTED** [review]
Great work. Just one detail. Could we improve the look of the info icon a little bit? I would consider to display the info icon not surrounded by a button.
Thus, the icon as it is, but in blue and remove the outer button boundary. What do you think?

File: libs/api/src/lib/graphql/permissions.ts [hunk 2]
```diff
@@ -708,7 +708,6 @@ export const EditorPermissions: Permission[] = [
   CanGetPeers,
   CanGetPeerProfile,
   CanLoginEditor,
-  CanGetSettings,
```
Comment (penk85) [inline libs/api/src/lib/graphql/permissions.ts:711]: I think we should still keep this permission in the api. Just remove the qualifying permission of "CAN_GET_SETTINGS" from the settings page .

**Review by penk85 — CHANGES_REQUESTED** [review]
Looks great. Just I'd keep the permission in the api and update in the editor. Both the settingList and the settings tab in the base sidebar.

File: libs/api/src/lib/graphql/permissions.ts (hunk 2 above)
Comment (penk85) [reply libs/api/src/lib/graphql/permissions.ts:711]: Ah you're right. This makes sense.

**Review by penk85 — APPROVED** [review]

**Review by michael-scheurer — APPROVED** [review]

**Pull request merged**